### ISSUE TITLE
Validate: Ethereum Profile Login Button

### ISSUE THREAD

**Issue by kamescg** [3box/3box#254]
# Ethereum Profile Login Button
**Why:** Connect Humans with Web3
**What:** A Universal Ethereum Profile Login to manage a 3Box Database.  The Portal will open a Human Focused Dashboard that creates a symbiotic relationship with any distributed application frontend: wrapping input/output methods to and from private databases and services owned by the user specific to that distributed application.
**How:** Embeddable universal digital identity login portal to connect with 3Box API methods.

### User Story
As a User I want to manage my 3Box Database from anywhere on the Internet. 

### Objective
Create an easily embeddable Universal Ethereum Profile Login that allows Universal access to the 3Box Database and Distributed Application ecosystem. The login button will wrap the 3Box API (current and future features) and enable websites to provide a custom login experience by allowing each distributed application to pass in `metadata` to contextualize the user experience.

### Acceptance Criteria
- Single line of Javascript embed code.
- Pass in metadata from dApp to login portal: style, views and privilege requests.
- Filter activity for distributed application specific requests.

![Login Buttons](https://ipfs.io/ipfs/QmdmUr7GunvHESXz9T4EgNrzxEUmeN5co4uf2q8UVZQCtw)
![Dashboard](https://ipfs.io/ipfs/QmTMYJtkLRF6Y9qAfa7LQ2sjySJMY785JMuS8gZRSqdk9U)

**Comment by KamesCG** [comment]
As I was gallivanting the streets last night, thinking about all the way Humans will start interacting with the Web3 it seemed evident the 3Box Ethereum Profiles solution is the likely candidate for becoming the Solution to replace standard Social Profile Logins for a number of critical reasons: privacy, interoperability and direct access to future Internet - Web3. It goes without saying, due to all the recent data breaches, emerging cross-protocol capabilities and difficult learning curve to adopt cryptography in day-to-day interactions it could be an optimal time to introduce a simple button that focuses on user privacy and abstracts away all the difficulties of Web3 in a single click.

Thus, I wanted to start the conversation regarding how do we help Web3 Developers quickly integrate Profiles (Identity) into their distributed applications in the shortest amount of steps possible. One could surmise a Universal Login button that creates a privacy preserving, but frictionless experience to managing an Ethereum Profile might be the path towards kickstarting a 3Box movement :)

In summary the Universal Login Button is probably not a 3Box product (maybe it is), but rather an invitation to simply discuss how to provide the highest value amount of value to the largest number of users, in the shortest amount of time that are currently in the Web3 space.

TL;DR: I got excited. Wanted to share. Created a placeholder for a conversation. #BuidlforUsers.

**Comment by jjperezaguinaga** [comment]
I [shared](https://twitter.com/jjperezaguinaga/status/1036935686291685376) a month ago the shocking fact we still don't have a secure way to say "Hey, 0x111...111 is me", where "me" is a social probable identity that ties my internet footprint. This is a crucial use case for many individuals and companies across the industry ("Hey this is Shane from Company A, please send the 200K invoice to this address that's totally a company address and not my way out before quitting this company and relocating to Colombia"), and we have yet to have found a common standard. I see in 3Box as an option.

I'm going to be in DevCon4 and Status.im Hackathon next week. Would be great to meet to discuss how to push 3Box forward to a more user friendly solution.

**Comment by oed** [comment]
@jjperezaguinaga We'll be at devcon! Let's meet there ☺️
